tutorial/header-files.md

```diff
@@ -24,6 +25,61 @@ print('number of threads is :', omp_get_num_threads())
 ```
 Pyccel can compile the Python file with the following command: `pyccel openmp.py --language fortran --flags="-fopenmp"`
```

Comment:
I mentioned this before. Are we sure that we cannot simply use this command,
```bash
$ pyccel openmp.py --language fortran --openmp
```
which should work with different compilers on different platforms?

Reply:
yes we can use this command for sure, it will work